src/sentry/static/sentry/app/components/events/contexts/trace/getTraceKnownDataDetails.tsx

```diff
@@ -0,0 +1,152 @@
+};
+
+function getUserKnownDataDetails(
+  data: TraceKnownData,
+  type: TraceKnownDataType,
+  event: Event,
+  organization: Organization
+): Output | undefined {
+  switch (type) {
+    case TraceKnownDataType.TRACE_ID: {
+      const traceId = data.trace_id || '';
+
+      if (!traceId) {
+        return undefined;
+      }
+
+      const dateCreated = moment(event.dateCreated).valueOf() / 1000;
+      const pointInTime = event?.dateReceived
+        ? moment(event.dateReceived).valueOf() / 1000
+        : dateCreated;
+
+      const {start, end} = getTraceDateTimeRange({start: pointInTime, end: pointInTime});
+
+      const orgFeatures = new Set(organization.features);
+
+      const traceEventView = EventView.fromSavedQuery({
+        id: undefined,
+        name: `Events with Trace ID ${traceId}`,
+        fields: ['title', 'event.type', 'project', 'trace.span', 'timestamp'],
+        orderby: '-timestamp',
+        query: `trace:${traceId}`,
+        projects: orgFeatures.has('global-views') ? [] : [Number(event.projectID)],
+        version: 2,
+        start,
+        end,
+      });
+      return {
+        subject: t('trace_id'),
+        value: (
+          <ButtonWrapper>
+            <pre className="val">
+              <span className="val-string">{traceId}</span>
+            </pre>
+            <StyledButton
+              size="xsmall"
+              to={traceEventView.getResultsViewUrlTarget(organization.slug)}
+            >
+              {t('Search by Trace')}
+            </StyledButton>
+          </ButtonWrapper>
+        ),
+      };
+    }
+
+    case TraceKnownDataType.SPAN_ID: {
+      return {
+        subject: t('span_id'),
+        value: data.span_id || '',
+      };
+    }
+
+    case TraceKnownDataType.PARENT_SPAN_ID: {
+      return {
+        subject: t('parent_span_id'),
+        value: data.parent_span_id || '',
+      };
+    }
+
+    case TraceKnownDataType.OP_NAME: {
+      return {
+        subject: t('op'),
+        value: data.op || '',
+      };
+    }
+
+    case TraceKnownDataType.STATUS: {
+      return {
+        subject: t('status'),
+        value: data.status || '',
+      };
+    }
+
+    case TraceKnownDataType.TRANSACTION_NAME: {
+      const eventTag = event?.tags.find(tag => {
+        return tag.key === 'transaction';
+      });
+
+      if (!eventTag) {
+        return undefined;
+      }
+
+      const transactionName = eventTag.value || '';
+
+      if (!transactionName) {
+        return undefined;
+      }
+
+      const to = transactionSummaryRouteWithQuery({
+        orgSlug: organization.slug,
+        transaction: transactionName,
+        projectID: event.projectID,
+        query: {},
+      });
+
+      return {
+        subject: t('transaction'),
+        value: (
+          <ButtonWrapper>
+            <pre className="val">
+              <span className="val-string">{transactionName}</span>
+            </pre>
+            <StyledButton size="xsmall" to={to}>
```

Comment:
I think you need a Feature gate on this as mm2 plans will have discover2 but not have performance views.

Reply:
should be addressed in https://github.com/getsentry/sentry/pull/19078/commits/cb8a8a503e4e3350c3d19ea10730559855bc399d